insights/parsers/tests/test_bond.py

```diff
@@ -87,12 +113,12 @@ def test_bond_class():
     assert bond_obj.bond_mode == '4'
     assert bond_obj.partner_mac_address == "00:00:00:00:00:00"
     assert bond_obj.aggregator_id == ['3', '3', '2']
+    assert bond_obj.xmit_hash_policy == "layer2"
 
     bond_obj = Bond(context_wrap(BONDINFO_CORRUPT, CONTEXT_PATH))
     assert not bond_obj.bond_mode
     assert bond_obj.slave_interface == []
+    assert not bond_obj.xmit_hash_policy
 
-    with pytest.raises(ParseException) as exc:
```

Comment:
@vishwanathjadhav - why you removed this test case?

Reply:
restored the test case please check